wca-regulations.md

```diff
@@ -413,11 +413,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A7a1) If the judge finds that the puzzle is solved, they call "OKAY".
         - A7a2) If the judge assigns any penalties, they call "PENALTY".
         - A7a3) If the result is DNF, the judge calls "DNF".
-    - A7b) The judge records the result on a score sheet.
+    - A7b) The judge records the result on a score sheet and signs next to the result.
```

Comment:
I'd say we kindly ask Luis to change the scoresheet :)

I asked him once but can't remember the reasons he gave back then.
@Luis-J-Ianez ?